pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -412,8 +418,18 @@ func (w *wlReconciler) reconcileGroup(ctx context.Context, group *wlGroup) (reco
 
 			wlPatch := workload.BaseSSAWorkload(group.local)
 			workload.SetAdmissionCheckState(&wlPatch.Status.AdmissionChecks, *acs, w.clock)
+
+			// Set the cluster name to the reserving remote and clear the nominated clusters.
+			wlPatch.Status.ClusterName = &reservingRemote
+			wlPatch.Status.NominatedClusterNames = []string{}
```

Comment:
```suggestion
			wlPatch.Status.NominatedClusterNames = nil
```

We do not distinguish between an empty and nil slice.

Reply:
Done